### PR TITLE
Changed docs for manifest multi lang

### PR DESCRIPTION
**Pull request by nenadb97**

### CODE REVIEW

File: docs/general/manifest-api.md [hunk 1]
```diff
@@ -81,11 +87,24 @@ curl -X PATCH "https://api.socialhub.io/manifest?accesstoken=eyJhbGciOiJIUzI1NiI
 |-----------------|-----------------------------------------------------------|
 | `type`          | Type of the Ticket Action. At the moment we support `reply`, `template_reply` and `button` actions. There may be multiple actions of the same type. |
 | `id`            | Identifier of the Action. Each Action within a manifest must have a different identifier. Pattern regular expression: `^[a-zA-Z0-9-_]{1,256}$` |
-| `label`         | Human readable button label for this action. May be up to 256 characters long but should be as short as possible. |
+| `label`         | Configuration of human readable title of the option which will be shown to the user as button label. |
```
Comment (mikhalev-im) [inline docs/general/manifest-api.md]: Sound a bit confusing, can we maybe change it to something more simple:

`Button labels for different locales. Locale is selected depending on user settings.`

File: docs/general/manifest-api.md [hunk 2]
```diff
@@ -81,11 +87,24 @@ curl -X PATCH "https://api.socialhub.io/manifest?accesstoken=eyJhbGciOiJIUzI1NiI
...
 | `config`        | Configuration options for this Ticket Action. |
 | `attachments`   | If set (`{}`) for Ticket Actions of type `reply`, it's possible to attach files for the reply. At a later point you'll be able to specify a file schema to control what kind of attachments are allowed – for now there is no restriction an all checks should happen on the Integration's end. |
 | `options`       | Array of dropdown options for the action. At the moment supported for the `reply` action. If `options` are set for an action, the user will have to choose one from the dropdown before creating the reply. |
 
+#### `inbox.ticketActions[].label`
+
+| Field           | Description                                               |
+|-----------------|-----------------------------------------------------------|
+| `en`            | Human readable title of the option which will be shown to the user as button label in English. |
```
Comment (mikhalev-im) [inline docs/general/manifest-api.md]: Lets not mix `title` / `label` and `option` / `button` in one sentence

I guess something like this should be enough:
`Human-readable label for English locale.`

File: docs/general/manifest-api.md [hunk 3]
```diff
@@ -81,11 +87,24 @@ curl -X PATCH "https://api.socialhub.io/manifest?accesstoken=eyJhbGciOiJIUzI1NiI
...
+| `de`            | Human readable title of the option which will be shown to the user as button label in German language. |
```
Comment (mikhalev-im) [inline docs/general/manifest-api.md]: `Human-readable label for German locale.`

File: docs/general/manifest-api.md [hunk 4]
```diff
@@ -81,11 +87,24 @@ curl -X PATCH "https://api.socialhub.io/manifest?accesstoken=eyJhbGciOiJIUzI1NiI
...
+
+#### `inbox.ticketActions[].options[].label`
+| Field           | Description                                               |
+|-----------------|-----------------------------------------------------------|
+| `en`            | Human readable title of the option which will be shown to the user as button label in English. |
```
Comment (mikhalev-im) [inline docs/general/manifest-api.md]: Please adjust similar to above

File: docs/general/manifest-api.md [hunk 5]
```diff
@@ -121,14 +140,21 @@ curl -X PATCH "https://api.socialhub.io/manifest?accesstoken=eyJhbGciOiJIUzI1NiI
 | Field           | Description                                                  |
 |-----------------|--------------------------------------------------------------|
 | `id`            | String identifier for this sidebar Tab. |
-| `label`         | Human readable label to display for the sidebar Tab. |
+| `label`         | Configuration of human readable title of the option which will be shown to the user as button label. |
```
Comment (mikhalev-im) [inline docs/general/manifest-api.md]: Please adjust similar to above

File: docs/general/manifest-api.md [hunk 6]
```diff
@@ -121,14 +140,21 @@ curl -X PATCH "https://api.socialhub.io/manifest?accesstoken=eyJhbGciOiJIUzI1NiI
...
 The `flatListWithoutRoot` tree builder simply displays all Tickets in the right sidebar that share the same Root-Ticket excluding the Root-Ticket itself.
 
 `flatListWithRoot` tree builder displays all Tickets with the same Root-Ticket including the Root-Ticket.
 Root-Ticket in the sidebar will have some additional actions like `Show unread Tickets in the Inbox`.
 
+#### `inbox.rightSidebar[].label`
+
+| Field           | Description                                               |
+|-----------------|-----------------------------------------------------------|
+| `en`            | Human readable title of the option which will be shown to the user as button label in English. |
```
Comment (mikhalev-im) [inline docs/general/manifest-api.md]: Please adjust similar to above

**Review by mikhalev-im — CHANGES_REQUESTED** [review]
There is also a [swagger](https://github.com/socialhubio/socialhub-dev/blob/master/swagger.yaml) file in this repo that you need to adjust.

And please add a new entry to the [changelog](https://github.com/socialhubio/socialhub-dev/blob/master/docs/changelog.md).

File: docs/changelog.md [hunk 7]
```diff
@@ -6,6 +6,21 @@ sidebar_label: API Changelog
 
 ⚠ Consider subscribing to our [**API Newsletter**](http://eepurl.com/g2EiC1) to be notified about upcoming API changes in the future.
 
+## Version 1.12
+The following changes to the API are scheduled to be deployed in **calendar week 49**.
```
Comment (mikhalev-im) [inline docs/changelog.md]: Please keep the spacing consistent, for example, add an empty line after headers (`##` and `###`)

File: docs/changelog.md [hunk 8]
```diff
@@ -6,6 +6,21 @@ sidebar_label: API Changelog
...
+### New: Multi language support for labels in Manifest 
+Ticket action label, ticket action option label and right sidebar label will have multiple language configuration.
+
+New label object format will be mandatory for manifest API `PATCH` requests:
+
+`label: {
```
Comment (mikhalev-im) [inline docs/changelog.md]: Maybe multiline code block fits better here (```js)

```js
label: {
    en: 'Reply',
    de: 'Antworten'
}
```

**Review by schlagbauer — APPROVED** [review]
Adjusted some smaller things but it is ready to be merged now

**Pull request merged**